twilio/tests/test.bal

```diff
@@ -32,22 +32,31 @@ configurable string toNumber = os:getEnv("SAMPLE_TO_MOBILE");
 configurable string test_message = os:getEnv("SAMPLE_MESSAGE");
 configurable string fromWhatsappNumber = os:getEnv("SAMPLE_WHATSAPP_SANDBOX");
 configurable string twimlUrl = os:getEnv("SAMPLE_TWIML_URL");
+configurable string twilioAPIKey = os:getEnv("twilioAPIKey");
+configurable string twilioAPISecret = os:getEnv("twilioAPISecret");
 
 ConnectionConfig twilioConfig = {
-    accountSId: twilioAccountSid,
-    authToken: twilioAuthToken
+    authConfig: {
+        accountSId: twilioAccountSid,
+        authToken: twilioAuthToken
+    }
 };
+
+// ConnectionConfig twilioConfig = {
+//     authConfig: {
+//             accountSId: twilioAccountSid,
+//             apiKey: twilioAPIKey,
+//             apiSecret: twilioAPISecret
+//     }
+// };
```

Comment:
Can't we remove this?

Reply:
This was added to be used in API key-based authenication tests. Otherwise, we have to rewrite this again.